src/css/bootstrap-graphdb-theme.css

```diff
@@ -1313,6 +1313,17 @@ footer.footer .container-fluid.main-container p {
     content: "\e920";
 }
 
+.tab-error:after {
+    font-family: 'icomoon', sans-serif !important;
+    font-size: 1em;
+    vertical-align: middle;
+    line-height: 1;
+    position: absolute;
+    left: 0.1em;
+    content: "\e920";
+    color: var(--onto-orange);
```

Comment:
Shoundn't it be some red color or this is what we use in the WB for errors?

Reply:
I couldn't find a constant for errors. I can create a new one, and we can use it in the future."

Reply:
No. You shouldn't. Just use the bootstrap's danger color. There is an overriden bootstrap theme which is used here.

Reply:
Done as @svilenvelikov suggested.